Destiny.ScrimTracker.Logic/Destiny.ScrimTracker.Logic.csproj

```diff
@@ -6,7 +6,6 @@
 
   <ItemGroup>
     <Folder Include="Adapters" />
-    <Folder Include="Models" />

```

Comment:
Can you explain this to me? I'm surprised that when we add the `Models` folder, we delete this line.

Reply:
This was a woopsie on my part